src/main/java/pl/gispartner/ResourceManagerApp/persistance/ResourceRepository.java

```diff
@@ -0,0 +1,11 @@
+package pl.gispartner.ResourceManagerApp.persistance;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import pl.gispartner.ResourceManagerApp.model.ResourceEntity;
+
+@Repository
+public interface ResourceRepository extends JpaRepository<ResourceEntity, Long> {
```

Comment:
fyi - jesli rozszerzasz JpaRepository to taka klasa automatycznie jest czytana przez kontekst springa i @Repository nie jest niezbedne, mimo to fajnie jest to zostawic dla przejrzystosci kodu